Scripts/Mend SCA/get-malicious-packages.sh

```diff
@@ -1,16 +1,27 @@
 #!/bin/bash
 # Prerequisites:
 # apt install jq curl
-# WS_USERKEY - An administrator's userkey
-# WS_EMAIL - The administrator's email
-# WS_APIKEY - API Key for organization
-# WS_API_URL - e.g. https://api-saas.mend.io/api/v2.0
+# MEND_USER_KEY - An administrator's userkey
+# MEND_EMAIL - The administrator's email
+# WS_APIKEY - API Key for organization (optional)
+# MEND_URL - e.g. https://api-saas.mend.io/api/v2.0
```

Comment:
MEND_URL should just be https://saas.mend.io as shown in the CLI documentation, we need to create the correct api call url from this so the customers only need to add the MEND_URL that would be used for the CLI